src/components/App/App.js

```diff
@@ -55,6 +55,7 @@ class App extends Component {
       <Route exact path="/" render={() => <Form grabInput={this.grabInput} clearFilteredMovies={this.clearFilteredMovies}/>} />
       {this.state.isLoading && !this.state.error && <h2 className="loading">Loading...</h2>}
       {this.state.error && <h2 className="error">Sorry, there was an error. Please come back later.</h2>}
+      {this.state.searchedMovie && !this.state.filteredMovies.length && <h2 className='no-movies'>Sorry, no movies found. Please try again!</h2>}
```

Comment:
What a beautiful solution. If there was a search made. no matches in our filitered length then render this <h2> Beautifully done Blanche!